pkg/render/elasticsearchclusterconfig.go

```diff
@@ -94,6 +100,7 @@ func (c ElasticsearchClusterConfig) ConfigMap() *corev1.ConfigMap {
 			"clusterName": c.clusterName,
 			"replicas":    strconv.Itoa(c.replicas),
 			"shards":      strconv.Itoa(c.shards),
+			"flowShards":  strconv.Itoa(c.flowShards),
```

Comment:
What you've linked in another comment https://github.com/tigera/operator/blob/master/pkg/render/fluentd.go#L445-L446 also indicates that there is no separation of FLOW and other shards in fluentd. I think that means that this flowShards will do nothing.